server/spec/test_integration_WAPI_CANVAS.rb

```diff
@@ -330,7 +275,7 @@ def test_canvas_api_studenta_calendar_events_headers
 
     refute_nil @w
 
-    user='dlhaines'
+    user='XXX'
```

Comment:
:+1: Try a retired username, like the venerable `bjensen`, or an invalid one, like  `non_existent_user`.